pygsheets/worksheet.py

```diff
@@ -37,6 +37,8 @@ def __init__(self, spreadsheet, jsonSheet):
         self.client = spreadsheet.client
         self._linked = True
         self.jsonSheet = jsonSheet
+        # support column, row dimensions
+        self.jsonSheet['properties'].setdefault('dimensionProperties', {}) 
```

Comment:
It might not be a good idea to have `dimensionProperties` because `jsonSheet` is supposed to be same as  the [api response](https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets#SheetProperties). Also i don't think we need to store the dimension properties, we could have this in the `Cell` or in `Datarange`.